data_and_models/pipelines/ner/dvc.yaml

```diff
@@ -1,77 +1,200 @@
+    - ../../annotations/ner/annotations5_EmmanuelleLogette_2020-06-30_raw2_Disease.dev.spacy
+    outs:
+    - ../../models/ner/model1/model-best/
+  train_model_2:
+    cmd: >-
+      python -m spacy train
+      config.cfg
+      --output ../../models/ner/model2/
+      --paths.train ../../annotations/ner/annotations14_EmmanuelleLogette_2020-09-02_raw8_CellCompartmentDrugOrgan.train.spacy
+      --paths.dev ../../annotations/ner/annotations14_EmmanuelleLogette_2020-09-02_raw8_CellCompartmentDrugOrgan.dev.spacy
+      --paths.base_model en_ner_bionlp13cg_md
+    deps:
+    - Dockerfile
+    - config.cfg
+    - ../../annotations/ner/annotations14_EmmanuelleLogette_2020-09-02_raw8_CellCompartmentDrugOrgan.train.spacy
+    - ../../annotations/ner/annotations14_EmmanuelleLogette_2020-09-02_raw8_CellCompartmentDrugOrgan.dev.spacy
+    outs:
+    - ../../models/ner/model2/model-best/
+  train_model_3:
+    cmd: >-
+      python -m spacy train
+      config.cfg
+      --output ../../models/ner/model3/
+      --paths.train ../../annotations/ner/annotations6_EmmanuelleLogette_2020-07-07_raw4_TaxonChebi.train.spacy
+      --paths.dev ../../annotations/ner/annotations6_EmmanuelleLogette_2020-07-07_raw4_TaxonChebi.dev.spacy
+      --paths.base_model en_ner_craft_md
+    deps:
+    - Dockerfile
+    - config.cfg
+    - ../../annotations/ner/annotations6_EmmanuelleLogette_2020-07-07_raw4_TaxonChebi.train.spacy
+    - ../../annotations/ner/annotations6_EmmanuelleLogette_2020-07-07_raw4_TaxonChebi.dev.spacy
+    outs:
+    - ../../models/ner/model3/model-best/
+  train_model_4:
+    cmd: >-
+      python -m spacy train
+      config.cfg
+      --output ../../models/ner/model4/
+      --paths.train ../../annotations/ner/annotations9_EmmanuelleLogette_2020-07-08_raw6_CelltypeProtein.train.spacy
+      --paths.dev ../../annotations/ner/annotations9_EmmanuelleLogette_2020-07-08_raw6_CelltypeProtein.dev.spacy
+      --paths.base_model en_ner_jnlpba_md
+    deps:
+    - Dockerfile
+    - config.cfg
+    - ../../annotations/ner/annotations9_EmmanuelleLogette_2020-07-08_raw6_CelltypeProtein.train.spacy
+    - ../../annotations/ner/annotations9_EmmanuelleLogette_2020-07-08_raw6_CelltypeProtein.dev.spacy
+    outs:
+    - ../../models/ner/model4/model-best/
+  train_model_5:
+    cmd: >-
+      python -m spacy train
+      config.cfg
+      --output ../../models/ner/model5/
+      --paths.train ../../annotations/ner/annotations15_EmmanuelleLogette_2020-09-22_raw9_Pathway.train.spacy
+      --paths.dev ../../annotations/ner/annotations15_EmmanuelleLogette_2020-09-22_raw9_Pathway.dev.spacy
+      --paths.base_model en_ner_craft_md
+    deps:
+    - Dockerfile
+    - config.cfg
+    - ../../annotations/ner/annotations15_EmmanuelleLogette_2020-09-22_raw9_Pathway.train.spacy
+    - ../../annotations/ner/annotations15_EmmanuelleLogette_2020-09-22_raw9_Pathway.dev.spacy
     outs:
-    - ../../models/ner/model5
+    - ../../models/ner/model5/model-best/
```

Comment:
Yes. No.

Reply:
Then, should we really keep this structure ? I have the feeling it is already highly convoluted. Maybe we could rename those folders ?

Reply:
What would be the benefit?
I don't see it as a simplification. Indeed, we would move away from the standard.

Besides, a directory `model-last` (not tracked) is also created in `../../models/ner/model5/`.
So, moving `ner/model5/model-best/` to `ner/model5/` would be an issue for keeping this `model-last`.

As we have a clear goal to improve the NER performances, I think it makes sense to keep `model-last` for now.
Indeed, it could help us understand and tune the models.